polaris-react/src/components/Choice/Choice.scss

```diff
@@ -1,22 +1,93 @@
 @import '../../styles/common';
 
+// stylelint-disable polaris/conventions/polaris/custom-property-allowed-list -- Polaris component custom properties
 .Choice {
-  display: flex;
```

Comment:
This [was originally introduced](https://github.com/Shopify/polaris/pull/7252/files#diff-bbe59ef28a50dd337c927494c6fc327290d85b375b08a197b3d31e7426602679L4) to support `<ChoiceList`. However, it caused calculations to be off with the new bleed negative margins. Thankfully, `fill` meets the same need for `<ChoiceList`, so we're able to remove this (meaning `display: inline-flex` is used at all screen sizes... see below) 👍